src/ir_Hitachi.cpp

```diff
@@ -798,6 +799,9 @@ bool IRrecv::decodeHitachiAC(decode_results *results, uint16_t offset,
     if (nbits / 8 == kHitachiAc1StateLength &&
        !IRHitachiAc1::validChecksum(results->state, kHitachiAc1StateLength))
       return false;
+    if (nbits / 8 == kHitachiAc344StateLength &&
+        !IRHitachiAc3::hasInvertedStates(results->state, nbits / 8))
```

Comment:
pro / cons of using `kHitachiAc344StateLength` instead of `nbits / 8`

Reply:
Cut & paste fail. Fixed.